ansible_sdk/model/job_def.py

```diff
@@ -47,3 +48,4 @@ class AnsibleJobDef(_DataclassReplaceMixin):
     timeout: int = None
     role: str = ''
     roles_path: t.Optional[t.Union[str, list[str]]] = None
+    metrics_output_path: str = ''
```

Comment:
These can all use 3.10 type annotation syntax since we're using the `annotations` future, and should probably be init'd to `None` instead of `''`, eg: `role: str | None = None`, `roles_path: str | list[str] | None = None`

Reply:
`metrics_output_path` and `roles` should still be `None` instead of `''`, or is there a reason they need to be empty string?